content/notebooks/E1_qc-1.ipynb

```diff
@@ -6,8 +6,8 @@
    "source": [
     "# **Quantum programming I**\n",
     "\n",
-    "$\\newcommand{\\bra}[1]{\\langle#1|}$\n",
-    "$\\newcommand{\\ket}[1]{|#1\\rangle}$"
+    "$\\renewcommand{\\bra}[1]{\\langle#1|}$\n",
+    "$\\renewcommand{\\ket}[1]{|#1\\rangle}$"
```

Comment:
I changed this line as my local editor was complaining. This may conflict with https://github.com/ENCCS/qas2023/issues/3